packages/amplication-client/src/VersionControl/CommitsPage.tsx

```diff
@@ -26,6 +27,8 @@ const CommitsPage: React.FC<Props> = ({ match, moduleClass }) => {
   const { currentProject, currentWorkspace } = useContext(AppContext);
   const { commits, commitsError, commitsLoading } = useCommit();
   const commitIdx = getCommitIdx(commits, commitId);
+  const commitsBuilds =
+    commits.length > 0 && commitIdx > -1 && commits[commitIdx].builds?.length;
```

Comment:
I think we should use different states - one when there are no commits, and a second one when there are no, builds for a specific commit. 

it may be confusing for a user to click on "commit" and see nothing in the list even when there are commits to show. In this case we should show the list of commits and in the space of the builds we need to show a message "There are no builds to show"
We can continue with the current design and open an issue to complete the second state at a later stage.

Reply:
I commented about this on another issue

Reply:
https://github.com/amplication/amplication/issues/3710

Reply:
I looked at the issue, and if I understand correctly you agree with what I wrote above - so are we fixing this or opening a new issue to be fixed later?